lib/libutils/isoc/bget.c

```diff
@@ -601,8 +601,11 @@ void *bget(requested_size, poolset)
 #endif
 #endif
 
-    size += sizeof(struct bhead);     /* Add overhead in allocated buffer
-					 to size required. */
+    /* Add overhead in allocated buffer to size required. */
```

Comment:
The increment of `size` above could also cause a wrapping if I'm not mistaken.

Reply:
Yes, I think you're right. The above round up calculation of `size` may also cause the same issue.
I add an assert for overflow checking in the following commit.

Reply:
Use `ADD_OVERFLOW(...)` instead of assert?

Reply:
Yes, and we could actually drop the `assert()` too and only return `NULL` because this will never happen in practice unless there's sort of attack going on.

Reply:
Got it. It seems that return `NULL` makes more sense. I removed the asserts in the following update.